src/main/dotnet/Program.cs

```diff
@@ -1,104 +0,0 @@
-﻿using Microsoft.Build.BuildEngine;
-using Newtonsoft.Json.Linq;
-using System;
-using System.IO;
-using System.Linq;
-
-namespace ProjectFileParser
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            try
-            {
-                Parse(args[0]);
-            }
-            catch (Exception e)
-            {
-                Console.Error.WriteLine("Error during project file parsing: {0}", e);
-                Console.Error.WriteLine(e.StackTrace);
-                Environment.ExitCode = -1;
-            }
-        }
-
-        static void Parse(String file)
-        {
-            var obj = JObject.Parse(Console.In.ReadToEnd());
```

Comment:
You're no longer reading input params, are you ? I think paths will be wrongly computed if we don't pass the proper params (like Debug/Release, x86 etc.)